### ISSUE TITLE
Some non-blank notes appear as blank

### ISSUE THREAD

**Issue by tjluoma** [glushchenko/fsnotes#384]
For some reason, some notes are showing up as blank in FSNotes.

I took a short screencast to show what I'm seeing:

https://www.dropbox.com/s/7i2vnhq5mnbuk7f/FSNotes-Blank.mov?dl=0

/Applications/FSNotes.app:
	CFBundleShortVersionString: 2.3.1
	CFBundleVersion: 191

Mac OS X 10.13.6

**Comment by glushchenko** [comment]
Can you attach file? What encoding of file? UTF-8?

**Comment by tjluoma** [comment]
Sorry, I went through and fixed them all yesterday. But I can tell you that they all seemed to have UTF-8 characters, such as curly quotes or `•` instead of `*`.

If I find another one, I will attach.

**Comment by tjluoma** [comment]
[@dwrites - Remove useless icons from Safari s bookmarks bar.txt](https://github.com/glushchenko/fsnotes/files/2499091/dwrites.-.Remove.useless.icons.from.Safari.s.bookmarks.bar.txt)

Ah! I was able to restore one of them from Dropbox's versions. It is attached.

**Comment by glushchenko** [comment]
```
fluder@mb:~/Downloads$ file -I dwrites.-.Remove.useless.icons.from.Safari.s.bookmarks.bar.txt 
dwrites.-.Remove.useless.icons.from.Safari.s.bookmarks.bar.txt: text/plain; charset=iso-8859-1
```

iso-8859-1

**Comment by tjluoma** [comment]
Ah… so I wonder if these files were all iso-8859-1 files with UTF-8 characters in them.

They all appeared OK in BBEdit when I opened them, but I wonder if BBEdit was just automatically recognizing the proper encoding.

**Comment by glushchenko** [comment]
Fixed in 2.3.2
